### PR TITLE
Panel renderer

### PR DESCRIPTION
**Pull request by rolbk**
Implements a "popup panel" that shows basic information about a targeted entity.  
Displays Name, Distance, Health and Armor.
Also has a fancy popup animation. ^^

### CODE REVIEW

File: src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+        for (var entry : entityMap.entrySet()) {
+            var entity = entry.getKey();
+            var lcScale = entry.getValue();
+
+            if (lcScale.getLeft() == 0 && lcScale.getRight() == 0) {
+                continue;
+            }
+
+            var scale = bezierBlend(MathHelper.lerp(tickDelta, (float) lcScale.getLeft(), (float) lcScale.getRight()) / 5.0F);
+            System.out.println(entity.getName().getString() + " " + lcScale.getLeft() + " " + lcScale.getRight() + " " + scale);
```
Comment (nbes4) [inline src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java]: Should we remove this?

File: src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+                    entityMap.compute(entity, (k, v) -> {
+                        if (v == null) {
+                            return lookingAt ? p(0, 1) : p(0, 0);
+                        } else {
+                            if (lookingAt) {
+                                return p(v.getRight(), Math.min(v.getRight() + 1, 5));
+                            } else {
+                                return p(v.getRight(), Math.max(v.getRight() - 1, 0));
+                            }
```
Comment (nbes4) [inline src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java]: We could use the ternary operator here

**Review by nbes4 — APPROVED** [review]
LGTM!

File: src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java (hunk 1 above)
Comment (rolbk) [reply src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java]: Pls remove :D

File: src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java (hunk 2 above)
Comment (rolbk) [reply src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java]: Makes the line unnecessary long imo, but go ahead

Comment (nbes4) [reply src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java]: I see, I agree with the unnecassary long line. I'll recheck

File: src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java (hunk 1 above)
Comment (nbes4) [reply src/main/java/com/cgh/openglhf/openglhf/client/renderer/blaze3d/InfoPanelRenderer.java]: 👍

**Pull request merged**